### ISSUE TITLE
Instructions to run project locally needed

### ISSUE THREAD

**Issue by Restuta** [plusjade/nilclass.com#3]
I read in read.me that you are trying to be "product oriented" and instructions are not a priority, but this prevernts interested people like myself from contributing. Would you reconsider adding them please?

**Comment by plusjade** [comment]
@Restuta you are right. I have updated the repository to point to a working ruhoh version as well as a more comprehensive guide for getting started: https://github.com/plusjade/nilclass.com/#local-installation-and-development

Let me know how it goes!

commit: https://github.com/plusjade/nilclass.com/commit/3800f09550dfb0c79d6ebd255266acff1330a7d0
